code/game/objects/structures/false_walls.dm

```diff
@@ -141,6 +141,7 @@
 	return null
 
 /obj/structure/falsewall/examine_status(mob/user) //So you can't detect falsewalls by examine.
+	to_chat(user, "<span class='notice'>The outer plating is <b>welded</b> firmly in place.</span>")
```

Comment:
ha ha oh god reinforced false walls are just going to be 200% lying all the time aren't they

Reply:
I completely forgot about reinforced false walls, they actually do weld apart and don't deconstruct like rwalls.

What should we do, should they lie about how they can be deconstructed or be apparently fake when examined?

Reply:
I have nooooooooo idea. maybe make them decon like rwalls, but that might be a bit much.

Reply:
Done, reinforced false walls decon with wirecutters.